src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6845,7 +6845,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Do not compare function pointer values</value>
   </data>
   <data name="ERR_FunctionPointerTypesInAttributeNotSupported" xml:space="preserve">
-    <value>Using a function pointer type in a 'typeof' in an attribute is not supported.</value>
+    <value>Using a function pointer type in an attribute while in a 'default' of an enum or a 'typeof' is not supported.</value>
```

Comment:
>an attribute while in a 'default' of an enum or a 'typeof [](http://example.com/codeflow?start=44&length=57)

Could we simply say "in this context" instead of trying to enumerate all possibilities?

Reply:
Well, I thought providing details could be useful, but I agree this one got a bit too complex. Simplified.